doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -6,18 +6,28 @@ Vernacular commands
 .. _displaying:
 
 Displaying
---------------
+----------
 
 
 .. _Print:
 
-.. cmd:: Print @qualid
-   :name: Print
+.. cmd:: Print {? Term } @smart_qualid {? @univ_name_list }
+
+   .. insertprodn univ_name_list univ_name_list
+
+   .. prodn::
+      univ_name_list ::= @%{ {* @name } %}
+
+   Displays information about the object identified by :n:`@smart_qualid`.
```

Comment:
BTW, "information" is a bit vague. The main point of this command is to print the definitions of terms, even opaque terms (defined in proof mode with an ending `Qed`):

```coq
Theorem A : True.
easy.
Qed.
Print A.
(*
A = I
     : True
*)
```